pages/about.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About Us</title>
+    <link rel="stylesheet" href="/style/css/about_main.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <header>
+        <div class="menu-btn">
+            <span class="burger"></span>
+        </div>
+        <ul class="sm">
+            <li><a href="facebook.com" class="fa fa-facebook smicon"></a></li>
+            <li><a href="twitter.com" class="fa fa-twitter smicon"></a></li>
+            <li><a href="#" class="smicon">English</a></li>
+            <li><a href="#" class="smicon">My page</a></li>
+            <li><a href="#" class="smicon">Logout</a></li>
+        </ul>
```

Comment:
- [ ] Kindly remove the social media icons and those from the about page's mobile version, 😃.